tests/unit/components/app-popover/AppPopover.spec.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import AppPopover from '../../../../src/components/app-popover/AppPopover'
+import { vi } from 'vitest'
+
+describe('AppPopover component', () => {
+  const props = {
+    initialItems: <div data-testid='initial'>Initial items</div>,
+    showMoreElem: <button>Show more</button>,
+    initialItemsWrapperStyle: {
+      backgroundColor: 'red'
+    }
+  }
+  const childrenComponent = <div className='test'>Children</div>
+
+  vi.mock('@mui/material/Popover', () => {
+    return {
+      default: function MockOpenButton(props) {
+        return (
+          <>
+            <button
+              onClick={() => {
+                props.onClose()
+                console.log(props.open)
+              }}
+            >
+              Close
+            </button>
+
+            {props.open ? props.children : <div>Closed</div>}
+          </>
+        )
+      }
+    }
+  })
+
+  describe('hideElem is false', () => {
+    beforeEach(() => {
+      render(
+        <AppPopover {...props} hideElem={false}>
+          {childrenComponent}
+        </AppPopover>
+      )
+    })
+
+    it('Component is rendered correctly', () => {
```

Comment:
The description of the test should sound as a continuation of the `it`. Something like `it(should correctly render the component)`. Read [here](https://daily-dev-tips.com/posts/jest-test-vs-it/) more.